examples/tree_1d_dgsem/elixir_mhd_ryujones_shock_tube.jl

```diff
@@ -7,6 +7,34 @@ using Trixi
 gamma = 5/3
 equations = IdealGlmMhdEquations1D(gamma)
 
+"""
+    initial_condition_ryujones_shock_tube(x, t, equations::IdealGlmMhdEquations1D)
+
+Ryu and Jones shock tube test case for one dimensional ideal MHD equations. Contains
+fast shocks, slow shocks, and rational discontinuities that propagate on either side
+of the contact discontinuity. Exercises the scheme to capture all 7 types of waves
+present in the one dimensional MHD equations. It is the second test from Section 4 of
+- Ryu and Jones (1995)
+  Numerical Magnetohydrodynamics in Astrophysics: Algorithm and Tests
+  for One-Dimensional Flow
+  [DOI: 10.1086/175437](https://doi.org/10.1086/175437)
+!!! note
+    This paper has a typo in the initial conditions. Their variable `E` should be `p`.
+"""
+function initial_condition_ryujones_shock_tube(x, t, equations::IdealGlmMhdEquations1D)
+  # domain must be set to [0, 1], γ = 5/3, final time = 0.2
+  rho = x[1] <= 0.5 ? 1.08 : 1.0
+  v1 = x[1] <= 0.5 ? 1.2 : 0.0
+  v2 = x[1] <= 0.5 ? 0.01 : 0.0
+  v3 = x[1] <= 0.5 ? 0.5 : 0.0
+  p = x[1] <= 0.5 ? 0.95 : 1.0
+  inv_sqrt4pi = 1.0 / sqrt(4 * pi)
+  B1 = 2 * inv_sqrt4pi
+  B2 = x[1] <= 0.5 ? 3.6 * inv_sqrt4pi : 4.0 * inv_sqrt4pi
```

Comment:
Just out of curiosity, why `<=` and not `<` here? The Brio-Wu shock tube also uses `<`.

Reply:
@andrewwinters5000? You added this in 53fd46ad347eec5c14e119e4ec75b8b2671769d7

Reply:
The Ryu & Jones reference is slightly ambiguous in that they just define a left and right state but don't actually state things in terms of `<` versus `<=` so I just made a decision. The same is true for the Torrilhon shocktube test. For Brio & Wu I followed how FLASH does it and they use the `<`